core/src/main/java/be/orbinson/aem/dictionarytranslator/servlets/action/ExportDictionaryServlet.java

```diff
@@ -0,0 +1,130 @@
+package be.orbinson.aem.dictionarytranslator.servlets.action;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.request.RequestParameter;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.apache.sling.servlets.annotations.SlingServletResourceTypes;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import javax.servlet.Servlet;
+
+@Component(service = Servlet.class)
+@SlingServletResourceTypes(
+        resourceSuperType = "granite/ui/components/coral/foundation/form",
+        resourceTypes = "aem-dictionary-translator/servlet/action/export-dictionary",
+        methods = "POST")
+public class ExportDictionaryServlet extends SlingAllMethodsServlet {
+
+    public static final String KEY_HEADER = "KEY";
+    public static final String SLING_KEY = "sling:key";
```

Comment:
create a const class for all except `KEY_HEADER` and the ones present in https://sling.apache.org/apidocs/sling6/org/apache/sling/jcr/resource/JcrResourceConstants.html